src/impl_x86__base_implementation.inl

```diff
@@ -184,6 +184,7 @@ static int IsVendorByX86Info(const X86Info* info, const char* const name) {
   return equals(info->vendor, name, sizeof(info->vendor));
 }
 
+// TODO: Remove when deprecation period is over,
```

Comment:
what depreciation period?

Reply:
At least two releases, I'll add it to the comment.